SETUP.md

```diff
@@ -0,0 +1,27 @@
+# Development Environment Setup
+
+## Supported OS
+
+The example instructions are for Ubuntu 18.04 and for any other OS using ubuntu on a virtual machine (Windows: WSL, VirtualBox, Docker.

```

Comment:
What if we rewrite some of this to be focused toward windows but link to go install page for mac and Linux?

Reply:
Just merge it for now. If you (or someone else) can come up later with cli steps for mac, just add them. Similar with windows, but that seems to be high effort.

I added the link to the official download, so everyone should be happy.

(note that at this point, I'll most possibly not invest more time into this, you can take on if you want)

Reply:
(though maybe I give it tomorrow one more pass, I just had an idea to simplify further)